### PR TITLE
update(big.js): Missing optional rounding mode parameter in `toExponential`, `toFixed` and `toPrecision`

### PR DESCRIPTION
**Pull request by message**
Added missing optional rounding mode parameter in `toExponential`, `toFixed` and `toPrecision`, which was added since [v6.0.0](https://github.com/MikeMcl/big.js/blob/master/CHANGELOG.md#600)

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: [toFixed](https://github.com/MikeMcl/big.js/blob/master/big.js#L924-L939), [toExponential](https://github.com/MikeMcl/big.js/blob/master/big.js#L898-L911), [toPrecision](https://github.com/MikeMcl/big.js/blob/master/big.js#L976-L990)

### CODE REVIEW

**Comment by message** [comment]
I'm quite new to TypeScript and I want to ask, what will be if we removed `const` before `enum` declaration?

Right now both `Comparison` and `RoundingMode` defined as:
```ts
export const enum Comparison {}
export const enum RoundingMode {}
```
and I get `TS2748: Cannot access ambient const enums when the '--isolatedModules' flag is provided`

**Comment by message** [comment]
ping @nycdotnet @Ky6uk

File: types/big.js/index.d.ts [hunk 1]
```diff
@@ -253,9 +253,10 @@ export interface Big {
      * If dp is omitted, or is null or undefined, the number of digits after the decimal point defaults to the minimum number of digits necessary to represent the value exactly.
      *
      * @param dp Decimal places, 0 to 1e+6 inclusive
+     * @param rm Rounding mode: 0 (down), 1 (half-up), 2 (half-even) or 3 (up).
```
Comment (peterblazejewicz) [inline types/big.js/index.d.ts:256]:
```suggestion
     * @param [rm] Rounding mode: 0 (down), 1 (half-up), 2 (half-even) or 3 (up).
```

File: types/big.js/index.d.ts [hunk 2]
```diff
@@ -270,9 +271,10 @@ export interface Big {
      * This is also unlike Number.prototype.toFixed, which returns the value to zero decimal places.
      *
      * @param dp Decimal places, 0 to 1e+6 inclusive
+     * @param rm Rounding mode: 0 (down), 1 (half-up), 2 (half-even) or 3 (up).
```
Comment (peterblazejewicz) [inline types/big.js/index.d.ts:274]:
```suggestion
     * @param [rm] Rounding mode: 0 (down), 1 (half-up), 2 (half-even) or 3 (up).
```

File: types/big.js/index.d.ts [hunk 3]
```diff
@@ -285,9 +287,10 @@ export interface Big {
      * If sd is omitted, or is null or undefined, then the return value is the same as .toString().
      *
      * @param sd Significant digits, 1 to 1e+6 inclusive
+     * @param rm Rounding mode: 0 (down), 1 (half-up), 2 (half-even) or 3 (up).
```
Comment (peterblazejewicz) [inline types/big.js/index.d.ts:290]:
```suggestion
     * @param [rm] Rounding mode: 0 (down), 1 (half-up), 2 (half-even) or 3 (up).
```

File: types/big.js/test/big.js-module-tests.ts [hunk 4]
```diff
@@ -6,7 +6,7 @@
   Minor changes have been made such as adding variable definitions where required.
 */
 
-import { Big, RoundingMode } from 'big.js';
+import { Big, RoundingMode } from "big.js";
```
Comment (peterblazejewicz) [inline types/big.js/test/big.js-module-tests.ts:9]:
```suggestion
import { Big, RoundingMode } from 'big.js';
```
The file uses single ones

**Review by peterblazejewicz — APPROVED** [review]
@message this is ready to go, sorry for the dealy (owners first rule)
I've added some optial niceties, if you don't mind
LGTM!

**Comment by message** [comment]
Ready to merge

**Pull request merged**